spec/helpers/claims_helper_spec.rb

```diff
@@ -219,4 +219,22 @@
       expect(helper.student_loan_answers(claim)).to eq expected_answers
     end
   end
+
+  describe "#format_claim_reference" do
```

Comment:
When we move the helper to the other file we'll also want to move these specs to the corresponding spec file: `spec/helpers/admin/claims_helper_spec.rb`